libkpod/remove.go

```diff
@@ -0,0 +1,52 @@
+package libkpod
+
+import (
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/pkg/errors"
+)
+
+// ContainerRemove removes a r
+func (c *ContainerServer) ContainerRemove(container string, force bool) (string, error) {
+	ctr, err := c.LookupContainer(container)
+	if err != nil {
+		return "", err
+	}
+	ctrID := ctr.ID()
+
+	if err := c.runtime.UpdateStatus(ctr); err != nil {
+		return "", errors.Wrapf(err, "failed to update container state")
+	}
+
+	err = c.runtime.UpdateStatus(ctr)
+	if err != nil {
+		return "", errors.Wrapf(err, "could not update status for container %s", ctrID)
+	}
```

Comment:
On further investigation, it looks like the changes @mrunalp made earlier removed any call to update in ContainerRemove, so we can remove both of these calls safely